.cirrus.yml

```diff
@@ -315,6 +316,8 @@ task:
   << : *MAIN_TEMPLATE
   container:
     image: ubuntu:focal
+    cpu: 2
+    memory: 8G
```

Comment:
not sure if related, but those changes to this file can probably be split up, including the bump to jammy below

Reply:
> not sure if related, but those changes to this file can probably be split up

It will be no longer required after #27062.

> including the bump to jammy below

That is related to CMake, as Android's NDK 23 expects CMake 3.19+, while Ubuntu Focal provides [3.16](https://packages.ubuntu.com/focal/cmake).

Reply:
Ok. I mean that the NDK can be bumped regardless. I don't expect that anyone is building for android, and if they did, it should be fine to just require the latest Ubuntu LTS, which is Jammy, independent of cmake changes.

Reply:
> Ok. I mean that the NDK can be bumped regardless. I don't expect that anyone is building for android, and if they did, it should be fine to just require the latest Ubuntu LTS, which is Jammy, independent of cmake changes.

See #27063.